src/development/tools/sdk/upgrading.md

```diff
@@ -43,7 +45,7 @@ to download the Flutter SDK and dependent packages.
 For example:
 
 ```terminal
-$ flutter channel dev
+$ flutter channel stable
```

Comment:
Maybe this should be `beta` as changing from `stable` to `stable` won't do much.